### ISSUE TITLE
How do I get json file??

### ISSUE THREAD

**Issue by merif** [ashawkey/RAD-NeRF#12]
I'm going to train to make the video into a portrait. How do I get the files I need for learning?

For example, the Obama video. 
1. transforms_train(val).json
2. track_params.pt
3. additionally-need 

What more resources do I need to learn custom videos? There are too many obstacles to learning through scripts.
Thank you.

**Comment by ashawkey** [comment]
@merif  Hi, please check the [dara processing](https://github.com/ashawkey/RAD-NeRF#data-pre-processing) part in readme.

**Comment by merif** [comment]
The above problem occurs 
when i run this part {python data_utils/process.py data/<ID>/<ID>.mp4} in the Data-pre-processing part.

[Errno 2] No such file or directory: 'data/obama/track_params.pt'
The problem occurs when I finish Preparation of Data-pre-processing(Download face_tracking/3DMM files) and use Runscript as a custom learning video(already prepare custom trainning video).

When experimenting with the example Obama video, the above error occurred in the pre-processing part, so I used the data downloaded from the drive.
And, another error occurs if you use a pt that has already been downloaded through the Obama video drive.

No such file or directory: 'data/obama/transforms_train.json'

I faced the problem of not being able to find the file even though I followed the data pre-processing part


How do I get json and pt files??
Thank you for your quick response.
